patch/u-boot/u-boot-rockchip64/add_fdtfile_dt_overlays.patch

```diff
@@ -12,20 +12,21 @@ index 84462f3..3291f9f
  	select ENABLE_ARM_SOC_BOOT0_HOOK
  	imply CMD_FASTBOOT
 diff --git a/include/configs/rk3328_common.h b/include/configs/rk3328_common.h
-index 513237e..a1bcebd
+index 6be9ad7..f553425 100644
 --- a/include/configs/rk3328_common.h
 +++ b/include/configs/rk3328_common.h
-@@ -73,12 +73,13 @@
- 	"fdt_addr_r=0x01f00000\0" \
- 	"kernel_addr_r=0x02000000\0" \
- 	"ramdisk_addr_r=0x04000000\0"
+@@ -57,6 +57,7 @@
```

Comment:
This breaks Z28 Pro legacy for sure.
Do you have one to test?
If so I could prepare its move to mainline u-boot. If not... maybe we should ditch it?

Reply:
or simply move this patch under the board_rockpis ?

Reply:
This patch is actually not needed for rockpis only for rk3328 boards of which only z28 is left with legacy u-boot.
That's why I tolerated it failing for rockpis ;p

Still... do you have z28? We could switch it to mainline u-boot and ditch one of the legacy u-boots in rockchip64 and make some cleanup thanks to that :)

Reply:
Actually I do have it somewhere, yes :)

Reply:
I let myself modify the other PR.
I will try to prepare a z28 switching PR and if you find time (and the device ;p) at some point you could test it.